frontend/src/components/cluster/Overview.tsx

```diff
@@ -146,12 +136,14 @@ function EventsSection() {
           />,
         ],
       }}
+      defaultGlobalFilter={eventsFilter ?? undefined}
       data={events}
       errorMessage={Event.getErrorMessage(eventsError)}
       columns={[
         {
           label: t('Type'),
           getValue: event => event.involvedObject.kind,
+          size: 150,
```

Comment:
Is it possible to determine non-fixed sizes, like "min-content" or similar, as it is possible to do it with an HTML grid?

Reply:
The table has two strategies 'semantic' which uses simple th,td,tr and 'grid' which confusingly uses flexbox for each row. I tried doing something like what was before 'minmax(1fr, 150px)' but neither option has a straightforward way to do it. I'll take another look into that

Reply:
okay I can just completely override the layout and use grid for everything, just like SimpleTable. I've returned all the gridTemplate parameters in column so now it should behave the same way it used to.